docker/gitlab-docker-ci/Dockerfile

```diff
@@ -1,3 +1,33 @@
 FROM devdemisto/gitlab-content-ci:1.0.0.64455
+
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    curl \
+    liblzma-dev \
+    libbz2-dev \
+    libssl-dev \
+    libreadline-dev \
+    libsqlite3-dev \
+ && rm -rf /var/lib/apt/lists/*
+
+# Set environment variables for pyenv
+ENV PYENV_ROOT /root/.pyenv
+ENV PATH $PYENV_ROOT/bin:$PATH
+
+# Install Pyenv and other init steps
+RUN curl -s https://pyenv.run | bash \
+ && eval "$(pyenv init --path)" \
+ && eval "$(pyenv init -)" \
+ && eval "$(pyenv virtualenv-init -)"
+
+# Setup pyenv
+RUN pyenv install -s 2.7.18 && pyenv versions | grep 2.7.18 && \
+ pyenv install -s 3.8.15 && pyenv versions | grep 3.8.15 && \
+ pyenv install -s 3.9.16 && pyenv versions | grep 3.9.16 && \
```

Comment:
Why do we need those old versions? I don't think we even support it in our repo anymore

Reply:
3.9 i think there might be an image or two that are still stuck on it.
@yaakovpraisler should we remove 3.8 and down?

Reply:
Maybe it is worth to remove 3.9 as well (since we're in the process of removing the support for it.

Reply:
@JudahSchwartz We've discussed it in the KO with @kobymeir and it was decided that we do want them as it was before the migration to gitlab.